src/lib/loader.c

```diff
@@ -55,18 +55,21 @@ int timer_create(clockid_t clockid, struct sigevent *sevp, timer_t *timerid)
     __attribute__((weak));
 
 static void *s_libdl_handle = NULL;
+static __typeof(dlerror) *s_dlerror = NULL;
```

Comment:
```suggestion
static __typeof(dlerror) *s_dlerror = &dlerror;
```

Need to add at the top:
```
const char *dlerror(void) __attribute__((weak));
```